project/Dependencies.scala

```diff
@@ -1,9 +1,9 @@
 import sbt._
 
 object Dependencies {
-  private val zioVersion = "1.0.0-RC19"
+  private val zioVersion = "1.0.0-RC19-2"
 
-  lazy val awsDynamoDb = "com.amazonaws" % "aws-java-sdk-dynamodb" % "1.11.784"
+  lazy val awsDynamoDb = "com.amazonaws" % "aws-java-sdk-dynamodb" % "1.11.786"
```

Comment:
Very studious dependency management :)

Reply:
They seem to make a release roughly every half an hour but I'm keeping up